fdbsyncd/fdbsyncd.cpp

```diff
@@ -45,7 +47,31 @@ int main(int argc, char **argv)
             if (sync.getRestartAssist()->isWarmStartInProgress())
             {
                 sync.getRestartAssist()->readTablesToMap();
+                
+                steady_clock::time_point starttime = steady_clock::now();
+                while (!sync.isIntfRestoreDone())
+                {
+                    duration<double> time_span =
+                        duration_cast<duration<double>>(steady_clock::now() - starttime);
+                    int pasttime = int(time_span.count());
+
+                    if (pasttime++ > INTF_RESTORE_MAX_WAIT_TIME)

```

Comment:
why pasttime++?

Reply:
Looks like a bug which got introduced when I modified the code to use steady_clock. Will fix this